systemtests/src/main/java/com/github/streamshub/systemtests/utils/ConsoleUtils.java

```diff
@@ -0,0 +1,11 @@
+package com.github.streamshub.systemtests.utils;
+
+import com.github.streamshub.systemtests.Environment;
+
+public class ConsoleUtils {
+    private ConsoleUtils() {}
+
+    public static String getConsoleOperatorName() {
+        return Environment.CONSOLE_DEPLOYMENT_NAME + "-operator";
```

Comment:
Will it be always like this? I mean, same for OLM and Yaml manifests installation? For the YAML (Bundle) installation this will work, but the OLM deployment name contains version as well or?

Reply:
Well, technically yes. For YAML as you mention it is whatever i set. So i took the one used for OLM, this is as you say `prefix` for the operator deployment. In current version it's `streamshub-console-operator-v0.6.4`

Reply:
It depends on what you will use it. If for something like "give me deployment name by this prefix" then it will work all the time. If you will assume that this is the actual full deployment name, you will have issues with that. Just my two cents.

Reply:
So would you suggest handling all the operations by label? Or some different method?

Reply:
Okay lets make it more dynamic. In case of OLM install i can get the full name after the deployment process by its prefix.

Reply:
You can also have method that based on installation type, you will either return the constant (for Yaml and Helm possibly) and then for OLM it returns the deployment name based on prefix in Namespace (so you list all Deployments in the particular Namespace and return the matching Deployment).

Or, during the installation you can actually configure some object and then return it from the Setup class or something.

Reply:
> For OLM it returns the deployment name based on prefix in Namespace (so you list all Deployments in the particular Namespace and return the matching Deployment).

i like this one, hopefully my implementation is what you meant.